rails/app/views/admin/places/show.html.erb

```diff
@@ -0,0 +1,91 @@
+<%#
+# Show
+
+This view is the template for the show page.
+It renders the attributes of a resource,
+as well as a link to its edit page.
+
+## Local variables:
+
+- `page`:
+  An instance of [Administrate::Page::Show][1].
+  Contains methods for accessing the resource to be displayed on the page,
+  as well as helpers for describing how each attribute of the resource
+  should be displayed.
+
+[1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Show
+%>
+
+<% content_for(:title) { page.page_title } %>
+
+<header class="main-content__header" role="banner">
+  <h1 class="main-content__page-title">
+    <small><%= t("place") %></small><br />
+    <%= content_for(:title) %>
+  </h1>
+
+  <div>
+    <%= link_to(
+      t("administrate.actions.edit_resource", name: page.page_title),
+      [:edit, namespace, page.resource],
+      class: "button",
+    ) if valid_action?(:edit) && show_action?(:edit, page.resource) %>
+  </div>
+</header>
+
+<section class="main-content__body main-content__body--split">
+  <% if page.resource.photo.attached? %>
+    <%= image_tag url_for(page.resource.photo) %>
+  <% else %>
+    <%= image_tag asset_path("speaker.png") %>
+  <% end %>
+  <dl>
+    <dt class="attribute-label">
+      <%= t("helpers.label.#{page.resource_name}.#{:name}") %>
+    </dt>
+    <dd class="attribute-data">
+      <%= page.resource.name %>
+    </dd>
+    <dt class="attribute-label">
+      <%= t("helpers.label.#{page.resource_name}.#{:description}") %>
+    </dt>
+    <dd class="attribute-data">
+      <%= page.resource.description %>
+    </dd>
+    <dt class="attribute-label">
+      <%= t("helpers.label.#{page.resource_name}.#{:region}") %>
+    </dt>
```

Comment:
haha this is why i hate adminstrate. carry on!


(like, the problems it solves, so much easier for barrier to entry with CRUD vanilla rails)